plugins/monitoring/app/views/monitoring/application/_tabs.html.haml

```diff
@@ -34,7 +34,7 @@
         %span.loading-place
   - else 
     %li{role: 'presentation'}
-      = link_to 'Notification Methods', plugin('monitoring').notification_methods_path()
+      = link_to 'Notification Channels', plugin('monitoring').notification_methods_path()
```

Comment:
that was a wish from joachim,  "Notification Methods" should become "Notification Channels"